bin/project

```diff
@@ -85,11 +90,15 @@ if [ "$1" = "rm" ]
     else
        _rmproject "$2" "$3"
        # TODO if active project == removed what to do? _changeproject "$2"
+       _deactivate_project
         exit 0
     fi
 fi
 
-
+if [ "$1" = "deactivate" ]; then
+    _deactivate_project
+    exit 1
```

Comment:
I believe the exit code should be 0, signaling the command run correctly

Reply:
True that